doc/src/fix_bond_react.txt

```diff
@@ -70,7 +70,8 @@ changes will be referred to as 'reactions' throughout this
 documentation. Topology changes are defined in pre- and post-reaction
 molecule templates and can include creation and deletion of bonds,
 angles, dihedrals, impropers, bond-types, angle-types, dihedral-types,
```

Comment:
this should be "atom types", "bond types" and so on. the only use of these term with a hyphen is in the command summary, where the hyphen signifies, that the two words stand for a single parameter.